Doc/whatsnew/3.14.rst

```diff
@@ -295,6 +295,19 @@ For further information on how to build Python, see
 
    __ https://en.wikipedia.org/wiki/Tail_call
 
+.. attention::
+
+    This section previously reported a 9-15% geomean speedup. This number has since been
```

Comment:
Can we say "geometric mean" here?  I think geomean is too jargonny.

Reply:
:facepalm: thanks, but I automerged too quickly.